### ISSUE TITLE
Updates Take Incredibly Long

### ISSUE THREAD

**Issue by luciusbono** [luciusbono/Packer-Windows10#18]
Grabbing Windows updates takes an inordinately long amount of time for some reason, which can result in the build timing out and not creating an artifact. Just leaving this here as a placeholder.

**Comment by luciusbono** [comment]
It turns out that the updates are actually not the problem! A bug was introduced in Packer 0.12.1 that breaks the WinRM communicator with Virtualbox. I updated the `README.md` and made some other changes in this commit: https://github.com/luciusbono/Packer-Windows10/commit/e4c7a77eac6f01de19ce96c67eb7b17af18999a8

I did, however, uncover that my build was no longer compatible with newer versions of Virtualbox. That error has also been corrected, as well as some optimizations. 

Leaving this issue open until Packer 0.12.2 comes out and I can test.
